SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -178,6 +185,10 @@ public static void Main(string[] args)
 
                 var windowsNoEditorPath = Path.Combine(stagingDir, "WindowsNoEditor");
 
+                // force spatial networking to be enabled
+                var defaultGameIniPath = Path.Combine(windowsNoEditorPath, baseGameName, "Config", "DefaultGame.ini");
+                File.AppendAllText(defaultGameIniPath, defaultGameIniOverrideText);
```

Comment:
Could you pull this out into a function, since it's almost the same snippet 3 times?

Reply:
Yup, seems like a good idea. I've changed it now.